src/scripts/main.js

```diff
@@ -1,3 +1,310 @@
 'use strict';
 
-// write code here
+const NAME = 'Name';
+const POSITION = 'Position';
+const OFFICE = 'Office';
+const AGE = 'Age';
+const SALARY = 'Salary';
```

Comment:
suggest to make one constant as object and each constant will be as field of object
```
const FORM_FIELD = {
  NAME: 'name',
  ....
}